Formula/l/licensed.rb

```diff
@@ -55,13 +55,18 @@ def install
       gem 'licensed', '#{version}'
     EOS
 
+    # https://github.com/github/licensed/blob/main/test/fixtures/cli/.licensed.yml
     (testpath/".licensed.yml").write <<~EOS
-      name: 'test'
-      allowed:
-        - mit
+      name: test
+      ignored:
+        bundler:
+          # in CI, bundler is set up pretty wonky and isn't found at the expected location
+          - "bundler"
+
+      cache_path: .licenses_test
     EOS
 
     assert_match "Caching dependency records for test",
-                        shell_output(bin/"licensed cache")
+                        shell_output(bin/"licensed cache", 1)
```

Comment:
Expecting it to fail seems odd

Reply:
yes, I am gonna file an issue about this.

Reply:
looks like the licensed installed from `gem install licensed` output `0` for me

```
$ gem install licensed
...
$ licensed --version
4.4.0

$ licensed cache
Caching dependency records for test

$ echo $?
0
```